File: python/pneumatics.py
# configure pneumatics
descorer = DigitalOut(brain.three_wire_port.a)
match_loader = DigitalOut(brain.three_wire_port.b)

if controller.buttonL2.pressing():
    # cylinder up for descorer
    descorer.set(True)
elif controller.buttonL1.pressing():
    # cylinder down for descorer
    descorer.set(False)

if controller.buttonDown.pressing():
    # cylinder up for loader
    match_loader.set(True)
elif controller.buttonUp.pressing():
    # cylinder down for loader
    match_loader.set(False)

File: c++/pneumatics.cpp
// Configure pneumatics
digital_out descorer = digital_out(Brain.ThreeWirePort.A);
digital_out match_loader = digital_out(Brain.ThreeWirePort.B);

// Moving the descorer
if (Controller1.ButtonL2.pressing()) {
    // Descorer up
    descorer.set(true);
}
else if (Controller1.ButtonL1.pressing()) {
    // Descorer down
    descorer.set(false);
}

// Moving the loader
if (Controller1.ButtonDown.pressing()) {
    // Loader down
    match_loader.set(true);
}
else if (Controller1.ButtonUp.pressing()) {
    // Loader up
    match_loader.set(false);
}
